src/main/java/com/iliev/exercises/EvenlyDivisibleNumbers.java

```diff
@@ -0,0 +1,57 @@
+package com.iliev.exercises;
+public class EvenlyDivisibleNumbers {
+    public EvenlyDivisibleNumbers() {
+    }
+
+    public static int[] CalculateNumber(){
+
+      boolean smallestFound = false;
+      boolean biggestFound = false;
+        int[] minAndMaxNumbers = new int[2];
```

Comment:
Make sure you are following the same indentation in a class. You can auto format to be sure about that